src/aws/ecr_helpers.go

```diff
@@ -1,194 +0,0 @@
-package aws
```

Comment:
Most of this logic was extracted into into `application/deployer/push_application_images.go`, `application/deployer/push_service_image.go` and `aws/push_image_helper`